plugins/chat/assets/javascripts/discourse/components/chat/routes/channel-info-members.gjs

```diff
@@ -82,6 +86,7 @@ export default class ChatRouteChannelInfoMembers extends Component {
     if (this.filter?.length) {
       params.username = this.filter;
     }
+    this.updatedAt;
```

Comment:
Used to reload the list after removing a user.

This component uses a cached list from the server that is requested when filters change, to allow for lazy loading.

This is the best way I can think of to integrate those the concerns of lazy loading/filtering and reactivity when removing a user.

Reply:
Shouldn't the value of this be changed here or am I misunderstanding? So `this.updatedAt = Date.now()`?

Reply:
The value is changed in `#removeMember`, which smashes the cache. The value needs to be "used" here for Ember to know to invalidate the cache.